jms/1.7/modules/ROOT/pages/jms-listener.adoc

```diff
@@ -46,66 +79,79 @@ For example, if you have a priority Queue with Messages that need to be processe
 </flow>
 ----
 
-When a selector is configured, only the Messages matching it will be received, ignoring if any other Message is in the destination.
-
-== Mime Types and Encoding
-
-The JMS Connector does its best to auto determine a Message’s mime type (`contentType`) based on the `MM_MESSAGE_CONTENT_TYPE` property. However, there are cases in which that best guess is not enough, and you need first-hand knowledge of the Message’s content. In such cases, you can force that content type to a particular value by using the `inboundContentType` parameter.
+== Reply to Incoming Messages
 
-The same process works for encoding. By default, the connector will assume that the runtime’s default encoding matches the one in the Message if no other information is provided. You can set this by using the `inboundEncoding` parameter.
+When an incoming JMS message declares a `ReplyTo` header destination and the message is processed successfully without any error during the flow execution, the *On New Message* source automatically produces a response.
+In that case, when the flow completes the connector publishes the response to the destination specified in the *Destination* field.
 
+In the following example, you configure the response message section of the operation. All these configurations impact the outbound message:
```

Comment:
```suggestion
In the following example, you configure the response message section of the operation:
```
I think that configuring the response message includes the assumption of impacting the outbound/response message?